readthedocs/subscriptions/event_handlers.py

```diff
@@ -132,6 +132,16 @@ def subscription_updated_event(event):
         log.info("Re-enabling organization.", organization_slug=organization.slug)
         organization.disabled = False
 
+    if stripe_subscription.status not in (
+        SubscriptionStatus.active,
+        SubscriptionStatus.trialing,
+    ):
+        log.info(
+            "Organization disabled due its subscription is not active anymore.",
+            organization_slug=organization.slug,
+        )
+        organization.disabled = True
+
```

Comment:
We should move this to the `subscription_canceled` handler.

Reply:
We cannot move it there because it will only catch canceled subscriptions and not past due, invalid or similar ones.